main.js

```
@@ -1,17 +1,76 @@
-import './style.css';
-import {Map, View} from 'ol';
+import { Feature, Map, View } from 'ol';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)